.circleci/config.yml

```diff
@@ -751,6 +751,15 @@ workflows:
           requires:
             - hold-build-logspout
 
+      - hold-bake-gcp-dev-image:
```

Comment:
if we add this it should only be exposed on master. running on a branch could break how our dev image flow works